README.md

```diff
@@ -27,7 +27,25 @@ and `env_vars_deactivate.sh` to `etc/conda/deactivate.d/` to every environment.
 In the `scIB-R-integration` environment, R packages need to be installed manually.
 Activate the environment and install the packages `scran`, `Seurat` and `Conos` in R. `Conos` needs to be installed using R devtools.
 See [here](https://github.com/hms-dbmi/conos).
-
+Furthermore, to ensure that `rpy2` uses the correct R version, the `LD_LIBRARY_PATH` variable has to be set accordingly.
+First, determine the path of the environment using `rpy2`
+```
+echo $CONDA_PREFIX
+```
+Then deactivate the environment to prevent unwanted effects.
+Next, set the `LD_LIBRARY_PATH` variable in the `${CONDA_PREFIX}/etc/conda/activate.d/activate-r-base.sh`
+```
+LD_LIBRARY_PATH_OLD=$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH
+export LD_LIBRARY_PATH="${CONDA_PREFIX}/lib/R/lib/"
+```
+and the following lines in `${CONDA_PREFIX}/etc/conda/deactivate.d/deactivate-r-base.sh`
+```
+LD_LIBRARY_PATH=LD_LIBRARY_PATH_OLD
+export LD_LIBRARY_PATH
+unset LD_LIBRARY_PATH_OLD
+```
```

Comment:
you have this in the `activate_env.sh` and `deactivate_env.sh` files already... why not just refer to those faileds that they should be moved to the relevant directories?

Reply:
These scripts didn't work for me, as `gsl-config` doesn't seem to be installed on my system. However only `LD` needed to be set in order to get `rpy2` running correctly. Also, Daniel mentioned something about `r-essentials` solving some of the issues

Reply:
It might also be nicer to have all R-specific variables in the `${CONDA_PREFIX}/etc/conda/activate.d/activate-r-base.sh` and `${CONDA_PREFIX}/etc/conda/deactivate.d/deactivate-r-base.sh`, respectively

Reply:
Hmm... I think we can probably stick with env variables for now. If this doesn't work for you, could you just adapt the README to say there are 2 options to get this to work?

Reply:
Moved back to using the env files only and added more info on them in the README